bower.json

```diff
@@ -1,16 +1,31 @@
 {
-    "name": "angular-recaptcha",
-    "version": "2.1.1",
-    "keywords": ["angular", "captcha", "recaptcha", "vividcortex", "human", "form", "validation", "signup", "security", "login"],
-    "main": "release/angular-recaptcha.js",
-    "ignore": [
-        "**/.*",
-        "Gruntfile.js",
-        "src",
-        "node_modules",
-        "bower_components"
-    ],
-    "dependencies": {
-        "angular": "1.*"
-    }
+  "name": "angular-recaptcha",
+  "version": "2.1.1",
+  "keywords": [
+    "angular",
+    "captcha",
+    "recaptcha",
+    "vividcortex",
+    "human",
+    "form",
+    "validation",
+    "signup",
+    "security",
+    "login"
+  ],
+  "main": "release/angular-recaptcha.js",
+  "ignore": [
+    "**/.*",
+    "Gruntfile.js",
+    "src",
+    "node_modules",
+    "bower_components"
+  ],
+  "dependencies": {
+    "angular": "1.*"
+  },
+  "devDependencies": {
+    "angular-mocks": "~1.*",
+    "jquery": "~2.1.3"
```

Comment:
Yeah, seems like it.

But since most applications will use jQuery almost by default, they haven't got this error yet.

The weird thing is that it says it's an error because we're passing a selector, but we're not, we're using a class instead.

I also tried testing it with angular 1.2.28 too - no success.

We should open an issue to track it.